Added REX section from 6.9 to 6.10 per SATDOC-645 (#1212)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

----
# hammer capsule content synchronize --name ${{SmartProxy}} --skip-metadata-check true --async
----

> .Upgrading {SmartProxy} Servers Using Remote Execution in the {Project} web UI

Reviewer: ```suggestion
.Upgrading {SmartProxy} Servers Using Remote Execution in the {ProjectWebUI}
```

. Create a backup.
+
* On a virtual machine, take a snapshot.
* On a physical machine, create a backup.
+
For information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.

. Optional: If you made manual edits to DNS or DHCP configuration in the `/etc/zones.conf` or `/etc/dhcp/dhcpd.conf` files, back up the configuration files because the installer only supports one domain or subnet, and therefore restoring changes from these backups might be required.

. Optional: If you made manual edits to DNS or DHCP configuration files and do not want to overwrite the changes, enter the following command:
+
[options="nowrap"]
----

> # {satellite-installer --foreman-proxy-dns-managed=false \

Reviewer 2: ```suggestion
# {satellite-installer} --foreman-proxy-dns-managed=false \
```
Do you build your docs before submitting the PR?

Author: Every time. Sometimes I build several times.

Author: I must have overlooked that one in the last build.

--foreman-proxy-dhcp-managed=false
----

. In the {Project} web UI, navigate to *Monitor* > *Jobs*.

. Click *Run Job*.

. From the *Job category* list, select *Maintenance Operations*.

. From the *Job template* list, select *Capsule Upgrade Playbook*.

> . In the *Search Query* field, enter the host name of the Capsule.

Reviewer: ```suggestion
. In the *Search Query* field, enter the host name of the {SmartProxy}.
```

. Ensure that *Resolves to* shows *1 host*.

. In the *target_version* field, enter the target version of the Capsule.

. In the *whitelist_options* field, enter the whitelist options.

. For *Type of query*, click *Static Query* or *Dynamic Query* depending on the type of query.

> 

Reviewer: ```suggestion
```

There are no empty lines necessary between elements of a list.
